packages/account/src/Components/forms/idv-form.tsx

```diff
@@ -0,0 +1,309 @@
+            ? formatInput(example_format, current_input ?? e.target.value, '-')
+            : e.target.value;
+        setFieldValue(document_name, current_input, true);
+    };
+
+    return (
+        <React.Fragment>
+            <section className={classNames('idv-form', class_name)}>
+                <div className='details-form'>
+                    <div className='poi-form-on-signup__fields'>
+                        <div
+                            className={classNames('proof-of-identity__container', {
+                                'proof-of-identity__container--idv': hide_hint,
+                            })}
+                        >
+                            <div className={classNames('proof-of-identity__inner-container')}>
+                                <div className='proof-of-identity__fieldset-container'>
+                                    <fieldset className={classNames({ 'proof-of-identity__fieldset': !hide_hint })}>
+                                        <Field name='document_type'>
+                                            {({ field }: FieldProps) => (
+                                                <React.Fragment>
+                                                    <DesktopWrapper>
+                                                        <div className='document-dropdown'>
+                                                            <Autocomplete
+                                                                {...field}
+                                                                name='document_type'
+                                                                data-lpignore='true'
+                                                                error={touched.document_type && errors.document_type}
+                                                                autoComplete='off'
+                                                                type='text'
+                                                                label={localize('Choose the document type')}
+                                                                list_items={document_list}
+                                                                value={values.document_type.text}
+                                                                onBlur={(e: { target: HTMLInputElement }) => {
+                                                                    handleBlur(e);
+                                                                    if (!getDocument(e.target.value)) {
+                                                                        resetDocumentItemSelected();
+                                                                    }
+                                                                }}
+                                                                onChange={handleChange}
+                                                                onItemSelection={(item: Record<string, string>) => {
```

Comment:
@yauheni-deriv, Wouldn't it be better to give `item` an explicitly `type`, perhaps it can even be extracted and reused else where? 🤔  
```suggestion
                                                                onItemSelection={(item: { id: string; text: string; value: string; example_format: string; sample: image: string;  }) => {
```

Reply:
Will use a specific type